tests/perfs/compare.py

```diff
@@ -103,7 +103,7 @@ def load_config():
             <th>{},&nbsp;s</th>
             <th>{},&nbsp;s</th>
             <th>Ratio of speedup&nbsp;(-) or slowdown&nbsp;(+)</th>
-            <th>Relative difference (after_score&nbsp;−&nbsp;before_score) / before_score</th>
+            <th>Relative difference (after_score&nbsp;&minus&nbsp;before_score) / before_score</th>
```

Comment:
`&minus;`

Reply:
Another confused:
```
Ratio of speedup (-) or slowdown (+)
```

is it
```
Ratio of speedup (+) or slowdown (-)?
```
cc @ZhiHanZ

Reply:
> `&minus;`

Good catch.